bin/p2-dsctl/main.go

```diff
@@ -13,27 +13,31 @@ import (
 	"github.com/square/p2/pkg/kp"
 	"github.com/square/p2/pkg/kp/dsstore"
 	"github.com/square/p2/pkg/kp/flags"
+	"github.com/square/p2/pkg/labels"
 	"github.com/square/p2/pkg/logging"
 	"github.com/square/p2/pkg/manifest"
 	pc_fields "github.com/square/p2/pkg/pc/fields"
 	"github.com/square/p2/pkg/types"
 	"github.com/square/p2/pkg/util"
-	"k8s.io/kubernetes/pkg/labels"
+	klabels "k8s.io/kubernetes/pkg/labels"
 )
 
 const (
-	CmdCreate  = "create"
-	CmdGet     = "get"
-	CmdList    = "list"
-	CmdEnable  = "enable"
-	CmdDisable = "disable"
-	CmdDelete  = "delete"
-	CmdUpdate  = "update"
+	CmdCreate       = "create"
+	CmdGet          = "get"
+	CmdList         = "list"
+	CmdEnable       = "enable"
+	CmdDisable      = "disable"
+	CmdDelete       = "delete"
+	CmdUpdate       = "update"
+	CmdTestSelector = "test-selector"
+
+	SelectorNotSpecified = "thereisnowaysomeonewouldtypethis"
```

Comment:
I don't know if seeing this in a DS would tell me all I needed to know. Instead, we should try to make this self-documenting. How about: "defaultNullSelector", "null", or ø